### ISSUE TITLE
tensor to array

### ISSUE THREAD

**Issue by huodongqi** [tsc2017/inception-score#2]
with your help of this code, I can put it into my model to evaluate my images generated by GANs, but the problem confused me for almost a week is the input of the code is array, but I wanna input of the function in my model is 4D tensor, and I also wanna put the result of the IS to the tensorboard, I've tried several ways like put the transference of the function in Session to transfer tensor to array as the parameters of the function (but the error is about the initialization). can u help me handle this problems ?

**Comment by tsc2017** [comment]
Hi, in order to change the input and output to tensors, you can just use a line of code that looks like:
`tfgan.eval.classifier_score(images, classifier_fn = functools.partial(tfgan.eval.run_inception, output_tensor = 'logits:0'), num_batches = A_LARGE_NUMBER)` for each split. Since you are using a very large input tensor, make sure `A_LARGE_NUMBER` is large enough to aviod OOM. Then you can get the mean and standard deviation tensors across the splits with `
tf.nn.moments`.
